CONTRIBUTING.md

```diff
@@ -187,13 +189,90 @@ pip install -e ./
 
 It's going to install Addons in editable mode without compiling anything.
 You can modify source files and changes will be seen at the next Python 
-interpreter startup.
+interpreter startup. This command needs to be executed only once. 
+Now, anywhere on your system, if you do `import tensorflow_addons`, it's 
+going to import the code in this git repository.
+
+To undo this operation, for example, you want to later on 
+install TensorFlow Addons from PyPI, the release version, do:
+
+```
+pip uninstall tensorflow-addons
+```
```

Comment:
Would caveat this as being installed for a specific python environment. A best practice is that they use conda or virtualenv

Reply:
Hmmm, I'm used to work in a docker container so I never do that. For people working with a local python installation (outside docker) that's definitely the best course of action to make a virtual environnement indeed. I'll look into that.